docs/assets/css/netlify.less

```diff
@@ -30,6 +30,11 @@
 			margin-left: 0.4375em;
 		}
 	}
+
+	// Hide the admin buttons when printing
+	.respond-to-print({
```

Comment:
Very minor, but in our other "respond-to" utilities, we include a space inside the parens https://github.com/cfpb/design-system/blob/f4613641062df12454f3101e09232a7b94096f0e/packages/cfpb-layout/src/organisms/featured-content-module.less#L9